clang/test/Preprocessor/macro-reserved-attrs-cxx11.cpp

```diff
@@ -0,0 +1,100 @@
+// RUN: %clang_cc1 -triple x86_64-linux-gnu -std=c++11 -fsyntax-only -verify -pedantic %s -DTEST1
+// RUN: %clang_cc1 -triple x86_64-linux-gnu -std=c++11 -fsyntax-only -verify -pedantic %s -DTEST2
+// RUN: %clang_cc1 -triple x86_64-linux-gnu -std=c++11 -fsyntax-only -verify -pedantic %s -DTEST3
+// RUN: %clang_cc1 -triple x86_64-linux-gnu -std=c++11 -fsyntax-only -verify -pedantic %s -DTEST4
```

Comment:
It's worth noting that this diagnostic is on by default for everyone (the `-pedantic` isn't needed to issue the warning). Are we sure this issue is worth warning about by default or do we think users may find it to be too chatty?

Reply:
I've added `DefaultIgnore` for this diagnostic, to follow the behavior like `macro name is a reserved identifier` does. Is it ok?